static/js/message_edit.js

```diff
@@ -59,13 +59,21 @@ function is_topic_editable(message, edit_limit_seconds_buffer = 0) {
     );
 }
 
+function is_submessage(message) {
```

Comment:
I think a better name for this would be `has_submessage` or `is_widgetized_message`.  I prefer the latter, I think.

Reply:
I'm also a bit curious why we don't just use `is_widget_message` here.